### ISSUE TITLE
Add release pipeline stages

### ISSUE THREAD

**Issue by Reeposter** [panuoksala/streamdeck-azuredevops-plugin#17]
Could you please add possibility to filter release pipeline stages, in case of workflow I have we don't use separate branches for envs, but instead stages, it would be nice to have possibility to check if stage failed or to run it.

**Comment by panuoksala** [comment]
Hmm so you have like two different stages (lets say stage 1 and stage 2) and you would like to run build only for stage 2?

**Comment by Reeposter** [comment]
Yes, or just check if the latest deployment on that stage was a success

W dniu śr., 3.08.2022 o 14:59 Panu Oksala ***@***.***>
napisał(a):

**Comment by panuoksala** [comment]
OK, I will dig into it and see what kind of API support is for stages

**Comment by panuoksala** [comment]
Plugin is using this method to query releases. There doesn't seem to be support for querying stage specific deployment. We need to look for another way around it.
function getDeployments(project: string, definitionId?: number, definitionEnvironmentId?: number, createdBy?: string, minModifiedTime?: Date, maxModifiedTime?: Date, deploymentStatus?: DeploymentStatus, operationStatus?: DeploymentOperationStatus, latestAttemptsOnly?: boolean, queryOrder?: ReleaseQueryOrder, top?: number, continuationToken?: number, createdFor?: string, minStartedTime?: Date, maxStartedTime?: Date, sourceBranch?: string): Promise<Deployment[]>
